simpleclient_servlet/src/test/java/io/prometheus/client/filter/MetricsFilterTest.java

```diff
@@ -117,7 +117,7 @@ public Void answer(InvocationOnMock invocationOnMock) throws Throwable {
 
         final Double sum = CollectorRegistry.defaultRegistry.getSampleValue("foobar_baz_filter_duration_seconds_sum", new String[]{"path", "method"}, new String[]{path, HttpMethods.POST});
         assertNotNull(sum);
-        assertEquals(0.1, sum, 0.001);
+        assertEquals(0.1, sum, 0.01);
```

Comment:
Without this change the test was flaky and failed for me locally and failed in TravisCI based on system load.

I'll remove it from the PR, but I suspect it is needed in the long term.